core/src/main/java/hudson/util/CompoundEnumeration.java

```diff
@@ -38,7 +38,7 @@ public boolean hasMoreElements() {
     }
 
     @Override
-    public T nextElement() throws NoSuchElementException {
+    public T nextElement() {
```

Comment:
Creates an unused import for `NoSuchElementException`.

Reply:
Thank you for identifying this problem, we'll be looking very quickly at how to ensure that 'Indepth' doesn't reproduce this issue again.